modules/account-lib/test/unit/keyPair/index.ts

```diff
@@ -1,13 +1,13 @@
 /**
  * @prettier
  */
-import 'should';
+import should from 'should';
```

Comment:
is this needed?

Reply:
It's required by call 'should' directly. In this file is here

https://github.com/BitGo/BitGoJS/blob/c647d6cb36a13e2fbccedde16b506f4eb3bbd1f7/modules/account-lib/test/unit/keyPair/index.ts#L29